src/reporters/browser.js

```diff
@@ -38,10 +37,15 @@ export default class BrowserReporter {
     `
 
     // Log to the console
-    consoleLogFn(
-      '%c' + [tag, type].filter(Boolean).join(':'),
-      style,
-      ...logObj.args
-    )
+    if (typeof logObj.args[0] === 'string') {
+      consoleLogFn(
+        `%c${[tag, type].filter(Boolean).join(':')}%c ${logObj.args[0]}`,
```

Comment:
This part is repeated twice. Maybe we can refactor shared part in a variable outside of if conditon?
(`%c${[tag, type].filter(Boolean).join(':')})